mteb/models/cohere_v.py

```diff
@@ -360,3 +451,49 @@ def get_fused_embeddings(
     use_instructions=False,
     training_datasets=None,
 )
+
+cohere_embed_v4_multimodal_binary = ModelMeta(
+    loader=partial(cohere_v_loader, model_name="embed-v4.0", embedding_type="binary"),
+    name="Cohere/Cohere-embed-v4.0 (embedding_type=binary)",
+    languages=all_languages,
+    revision="1",
+    release_date="2024-12-01",
+    n_parameters=None,
+    memory_usage_mb=None,
+    max_tokens=128000,
+    embed_dim=1536,
+    license=None,
+    similarity_fn_name="cosine",
+    framework=[],
+    modalities=["image", "text"],
+    open_weights=False,
+    public_training_code=None,
+    public_training_data=None,
+    reference="https://docs.cohere.com/docs/embeddings",
+    use_instructions=False,
+    training_datasets=None,
+    adapted_from="Cohere/Cohere-embed-v4.0",
+)
+
+cohere_embed_v4_multimodal_int8 = ModelMeta(
+    loader=partial(cohere_v_loader, model_name="embed-v4.0", embedding_type="int8"),
+    name="Cohere/Cohere-embed-v4.0 (embedding_type=int8)",
```

Comment:
It seems like voyage uses `output_dtype`, I would probably keep these consistent.

> voyageai/voyage-3.5 (output_dtype=binary)